server/apps/users/models.py

```diff
@@ -21,7 +21,18 @@ def role(self):
         return None
 
     def get_full_name(self):
-        return f'{self.first_name}, {self.last_name}'
+        l = []
+        if self.first_name:
+            l.append(self.first_name)
+        if self.last_name:
+            l.append(self.last_name)
+        if not l:
+            return self.email
+        return ' '.join(l)
```

Comment:
Esto garda holani :P
```suggestion
       return ' '.join([
           l for l in [self.first_name, self.last_name] if l
       ]) or self.email
```